src/engine/source/hlp/src/hlp.cpp

```diff
@@ -96,6 +96,21 @@ static void executeParserList(std::string const &event, ParserList const &parser
                 }
                 break;
             }
+            case ParserType::Ts: {
+                TimeStampResult tsr;
+                if (parseTimeStamp(&eventIt, parser.endToken, tsr)) {
```

Comment:
Let´s discuss with the team if this is the expected result form.

Reply:
By now, lets use the same approach as **url** parser and load it in the map as:
result["timestamp.year"] = = tsr.year;

Reply:
ok